server/models/helpers/HTMLHelper.ts

```diff
@@ -1,20 +1,28 @@
-import { inline } from "css-inline";
+import { initWasm, inline } from "@css-inline/css-inline-wasm";
+import fs from "fs-extra";
 import env from "@server/env";
 
+let initialized = false;
+
 export default class HTMLHelper {
   /**
    * Move CSS styles from <style> tags to inline styles with default settings.
    *
    * @param html The HTML to inline CSS styles for.
    * @returns The HTML with CSS styles inlined.
    */
-  public static inlineCSS(html: string): string {
+  public static async inlineCSS(html: string): Promise<string> {
+    if (!initialized) {
+      const path = require.resolve("@css-inline/css-inline-wasm/index_bg.wasm");
```

Comment:
Why doesn't the module just deal with this implementation detail by the way? 🤔

Reply:
I think it would be nice to do this way indeed. I’ve recently switched the build implementation and mostly followed an example from [resvg-js](https://github.com/yisibl/resvg-js)